distributions/aws/examples/storage/efs/README.md

```diff
@@ -113,9 +133,39 @@ efs-claim   Bound    efs-pv   5Gi        RWX            efs-sc         5d16h
 In the following two sections we will be using this PVC to create a notebook server with Amazon EFS mounted as the workspace volume, download training data into this filesystem and then deploy a TFJob to train a model using this data. 
 
-## 5.3 Using EFS volume as workspace or data volume for a notebook server 
+## 3.0 Using EFS Storage in Kubeflow
+### 3.1 Changing the default Storage Class
+After installing Kubeflow, you can change the default Storage Class from `gp2` to the efs storage class you created during the setup. For instance, if you followed the automatic or manual steps, you should have a storage class named `efs-sc`. You can check your storage classes by running `kubectl get sc`.  
+  
+This is can be useful if your notebook configuration is set to use the default storage class (it is the case by default). By changing the default storage class, when creating workspace volumes for your notebooks, it will use your EFS storage class automatically. This is not mandatory as you can also manually create a PVC and select the `efs-sc` class via the Volume UI but can facilitate the notebook creation process and automatically select this class when creating volume in the UI. You can also decide to keep using `gp2` for workspace volumes and keep the EFS storage class for datasets/data volumes only.
+  
+To learn more about how to change the default Storage Class, you can refer to the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/#changing-the-default-storageclass).  
+  
+For instance, if you have a default class set to `gp2` and another class `efs-sc`, then you would need to do the following : 
+1. Remove `gp2` as your default storage class
+```
+kubectl patch storageclass gp2 -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+```
+2. Set `efs-sc` as your default storage class
+```
+kubectl patch storageclass efs-sc -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+```
+### 3.2 Creating and using EFS volumes as workspace or data volume for a notebook [Dynamic Provisioning]
+
+The following example works if you used the automatic setup with dynamic provisioning.
+
+Here is an example that illustrates how to create a PVC for your dataset in `ReadWriteMany` mode meaning it can be used by many notebooks at the same time as well as how to create a notebook with a workspace volume for the notebook data and how to specify that you want to use your dataset volume as data volume.
+Note that both of these volumes are created under the storage class `efs-sc` which represents the EFS storage class created earlier.
+  
+
+https://user-images.githubusercontent.com/26939775/153103745-70f93ac5-88f3-4387-b40d-b585fca80af4.mp4
```

Comment:
While I really like this screen capture linked, we need to keep the format for the static and dynamic provisioning the same - for now could you write as text please ? ok with adding the video as well, but ideally we should then include it for both static and dynamic.

Reply:
Do you think we could make another PR to improve the doc for static provisioning Instead ? 

We could make a quick video showing how to create a PV and PVC etc.